src/interfaces/analysis-options.interface.ts

```diff
@@ -43,6 +43,7 @@ export interface FileAnalysisOptions extends AnalysisContext {
   connection: ConnectionOptions;
   analysisOptions: AnalysisOptions;
   fileOptions: AnalyzeFoldersOptions;
+  reportOptions: ReportOptions;
```

Comment:
req: This needs to be optional, otherwise we are breaking the existing interface i.e. `reportOptions?: ReportOptions`